lib/litle.rb

```diff
@@ -27,6 +27,14 @@
 module ActiveMerchant #:nodoc:
   module Billing #:nodoc:
     class LitleGateway < Gateway
+      def initialize(options={})
+        if options.has_key?(:test_url)
```

Comment:
I still want to add the `unless options[:test_url].blank?` change since it explicitly deals with empty strings.